tsuru/client/auth.go

```diff
@@ -271,6 +274,128 @@ func (c *TeamList) Run(context *cmd.Context, client *cmd.Client) error {
 	return nil
 }
 
+type TeamInfo struct{}
+
+func (c *TeamInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "team-info",
+		Usage:   "team-info <team>",
+		Desc:    `Shows information about a specific team.`,
+		MinArgs: 1,
+	}
+}
+
+func (c *TeamInfo) Run(context *cmd.Context, client *cmd.Client) error {
+
+	context.RawOutput()
+	team := context.Args[0]
+	u, err := cmd.GetURL(fmt.Sprintf("/teams/%v", team))
+	if err != nil {
+		return err
+	}
+	request, err := http.NewRequest("GET", u, nil)
+	if err != nil {
+		return err
+	}
+	resp, err := client.Do(request)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode == http.StatusOK {
```

Comment:
Theorically the API won't return anything different from 200, 404 and 401 (according to your other PR - but you can add it), so you can check this behavior on the API's tests and in this case return some error along with the status to guarantee that the client won't be showing empty content.